**Importe** la librairie *pandas*<br>
**Importe** les données depuis le fichier csv avec la fonction *read_*<br>
**Initialise** une variable *df* qui contient l'objet dataframe<br>

In [ ]:
import pandas as pd
df=pd.read_csv('netflix_titles.csv')

**4. Affiche les 5 premières entrée du dataframe** via la variable df.

In [ ]:
df.head()

**4 bis. Affiche les 5 dernières entrée du dataframe** via la variable df.

In [ ]:
df.tail()

**5. Affichez les informations du DataFrame**<br> notamment le type d'index et
les colonnes, les valeurs non nulles et l'utilisation de la mémoire.

In [37]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 8807 entries, 0 to 8806
Data columns (total 12 columns):
 #   Column        Non-Null Count  Dtype 
---  ------        --------------  ----- 
 0   show_id       8807 non-null   object
 1   type          8807 non-null   object
 2   title         8807 non-null   object
 3   director      6173 non-null   object
 4   cast          7982 non-null   object
 5   country       7976 non-null   object
 6   date_added    8797 non-null   object
 7   release_year  8807 non-null   int64 
 8   rating        8803 non-null   object
 9   duration      8804 non-null   object
 10  listed_in     8807 non-null   object
 11  description   8807 non-null   object
dtypes: int64(1), object(11)
memory usage: 825.8+ KB


**6. Affiche la dimensionnalité du DataFrame.**<br>
*Variables* : Les variables sont les colonnes du DataFrame. Il y en a 12.<br>
*Observation* : Les observations sont les lignes du DataFrame. Il y en a 8807.

In [34]:
dimensions = df.shape
rows, columns = dimensions
print(rows, columns)

8807 12


**7. Affichez les colonnes du DataFrame.**


In [35]:
print(df.columns)

Index(['show_id', 'type', 'title', 'director', 'cast', 'country', 'date_added',
       'release_year', 'rating', 'duration', 'listed_in', 'description'],
      dtype='object')


8. **Affichez le type des différentes colonnes du DataFrame.**<br> Avez-vous des
données quantitatives (numériques) ? <br>Avez-vous des données
qualitatives (catégorielles) ? Si oui, lesquelles ?

In [33]:
# Filtrer les colonnes avec des types de données numériques
numeric_data = data_types.apply(pd.api.types.is_numeric_dtype)
print(numeric_data)

show_id         False
type            False
title           False
director        False
cast            False
country         False
date_added      False
release_year     True
rating          False
duration        False
listed_in       False
description     False
dtype: bool
